api/cloud_accounts_test.go

```diff
@@ -0,0 +1,284 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api_test
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/lacework/go-sdk/api"
+	"github.com/lacework/go-sdk/internal/intgguid"
+	"github.com/lacework/go-sdk/internal/lacework"
+)
+
+func TestCloudAccountTypeAwsCtsSqs(t *testing.T) {
+	assert.Equal(t,
+		"AwsCtSqs", api.AwsCtSqsCloudAccount.String(),
+		"wrong cloud account type",
+	)
+}
+
+func TestFindCloudAccountType(t *testing.T) {
+	cloudFound, found := api.FindCloudAccountType("SOME_NON_EXISTING_INTEGRATION")
+	assert.False(t, found, "cloud account type should not be found")
+	assert.Equal(t, 0, int(cloudFound), "wrong cloud account type")
+	assert.Equal(t, "None", cloudFound.String(), "wrong cloud account type")
+
+	cloudFound, found = api.FindCloudAccountType("AwsCtSqs")
+	assert.True(t, found, "cloud account type should exist")
+	assert.Equal(t, "AwsCtSqs", cloudFound.String(), "wrong cloud account type")
+
+	//cloudFound, found = api.FindCloudAccountType("GcpCfg")
```

Comment:
Could this be removed? or if its being kept for debug purposes, have a comment explaining it 👍

Reply:
@garyboyes This is the first Cloud Account of many to implement, I created the skeleton of how they should look like and started with one, the `AwsCtSqs` cloud account. I am just leaving these comments for later if anyone implements a new one, we will know where to plug new tests. (`GcpCfg` is one that we will add in the future)

Reply:
No worries @afiune, just wanted to be sure 😄